integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCommonMount.java

```diff
@@ -191,26 +215,237 @@ public void testCreateDomainUsingMultipleCommonMounts() {
+    List<CommonMount> commonMountList = domain1.getSpec().getServerPod().getCommonMounts();
+    assertFalse(commonMountList.isEmpty(), "CommonMount list is empty");
+    String searchString;
+    int index = 0;
+
+    CommonMount cmMount = commonMountList.stream()
+          .filter(commonMount -> oldImageName.equals(commonMount.getImage()))
+          .findAny()
+          .orElse(null);
+    assertNotNull(cmMount, "Can't find common Mount with Image name " + oldImageName
+        + "can't patch domain " + domainUid);
+
+    index = commonMountList.indexOf(cmMount);
+    searchString = "\"/spec/serverPod/commonMounts/" + index + "/image\"";
+    StringBuffer patchStr = new StringBuffer("[{");
+    patchStr.append("\"op\": \"replace\",")
+        .append(" \"path\": " + searchString + ",")
+        .append(" \"value\":  \"" + newImageName + "\"")
+        .append(" }]");
+    logger.info("Common Mount patch string: " +  patchStr);
+
+    V1Patch patch = new V1Patch((patchStr).toString());
+
+    boolean cmPatched = assertDoesNotThrow(() ->
+            patchDomainCustomResource(domainUid, domainNamespace, patch, "application/json-patch+json"),
+        "patchDomainCustomResource(Common Mount)  failed ");
+    assertTrue(cmPatched, "patchDomainCustomResource(common Mount) failed");
+
+    domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource after patching");
+    assertNotNull(domain1.getSpec(), domain1 + " /spec/serverPod is null");
+    assertNotNull(domain1.getSpec().getServerPod(), domain1 + " /spec/serverPod is null");
+    assertNotNull(domain1.getSpec().getServerPod().getCommonMounts(), domain1 + "/spec/serverPod/commonMounts is null");
+
+    //verify the new CommonMount image in the new patched domain
+    commonMountList = domain1.getSpec().getServerPod().getCommonMounts();
+    
+    String cmImage = commonMountList.get(index).getImage();
+    logger.info("In the new patched domain imageValue is: {0}", cmImage);
+    assertTrue(cmImage.equalsIgnoreCase(newImageName), "common mount image was not updated"
+        + " in the new patched domain");
+
+    //get current timestamp before domain rolling restart to verify domain roll events
+    OffsetDateTime timestamp = now();
+
+    // verify the server pods are rolling restarted and back to ready state
+    logger.info("Verifying rolling restart occurred for domain {0} in namespace {1}",
+        domainUid, domainNamespace);
+
+    assertTrue(verifyRollingRestartOccurred(podsWithTimeStamps, 1, domainNamespace),
+        String.format("Rolling restart failed for domain %s in namespace %s", domainUid, domainNamespace));
+  }
+
+  private Map getPodsWithTimeStamps(String adminServerPodName, String managedServerPrefix,
```

Comment:
have you checked if there are any utility methods? we may be doing this in multiple tests

Reply:
switched to common util